src/components/ftbot/TradeListNav.vue

```diff
@@ -11,21 +11,36 @@ const emit = defineEmits<{ 'trade-select': [trade: Trade] }>();
 
 const botStore = useBotStore();
 const selectedTrade = ref({} as Trade);
-const sortNewestFirst = ref(true);
+const sortDescendingOrder = ref(true);
+const sortMethod = ref('openDate');
+const sortMethodOptions = [
+  { text: 'Open date', value: 'openDate' },
+  { text: 'Profit %', value: 'profit' },
+];
 
 const onTradeSelect = (trade: Trade) => {
   selectedTrade.value = trade;
   emit('trade-select', trade);
 };
 
 const sortedTrades = computed(() => {
-  return props.trades
-    .slice()
-    .sort((a, b) =>
-      sortNewestFirst.value
-        ? b.open_timestamp - a.open_timestamp
-        : a.open_timestamp - b.open_timestamp,
-    );
+  if (sortMethod.value === 'profit') {
+    return props.trades
+      .slice()
+      .sort((a, b) =>
+        sortDescendingOrder.value
+          ? b.profit_ratio - a.profit_ratio
+          : a.profit_ratio - b.profit_ratio,
+      );
+  } else {
+    return props.trades
+      .slice()
+      .sort((a, b) =>
+        sortDescendingOrder.value
+          ? b.open_timestamp - a.open_timestamp
+          : a.open_timestamp - b.open_timestamp,
+      );
+  }
```

Comment:
```suggestion
  const field: keyof Trade = sortMethod.value === 'profit' ? 'profit_ratio' : 'open_timestamp';
  return sortDescendingOrder.value
    ? props.trades.slice().sort((a, b) => b[field] - a[field])
    : props.trades.slice().sort((a, b) => a[field] - b[field]);
```

 I think this can / should be simplified (a ton, actually).
 it'll also (accidentally, on purpose) improve performance over the previous version :D